pkg/docker/cockpit-atomic-storage

```diff
@@ -428,8 +424,10 @@ def cmd_monitor():
         if len(r) > 0:
             if mon.stdout.readline().decode().startswith("UDEV  "):
                 got_event = True
+        # we want to periodically all get_info() to update the usage
+        # information even without uevents
         # XXX - skip udev/lvm stuff on pure timeouts
-        info = get_info()
+        info = get_info(got_event)
```

Comment:
I called the (unused) function argument `got_uevent` to clarify its type (not just any kind of event, but an uevent from the kernel). The variable in the caller was already called `got_event`, but there it's clear from the context that it's about udev.

This is actually covered by tests, as in the first version I forgot to add the `False` to the `old_info = get_info()` call above, and it blew up in my face ☺.